samples/theia-hello-world-frontend-plugin/devfile.yml

```diff
@@ -0,0 +1,65 @@
+specVersion: 0.0.1
+name: theia-hello-world-plugin
+projects:
+  - name: che-theia-samples
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che-theia-samples.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
+    id: org.eclipse.che.editor.theia:1.0.0
+  - name: exec-plugin
+    type: chePlugin
+    id: che-machine-exec-plugin:0.0.1
+  - name: theia-dev
+    type: kubernetes
+    local: dev.yaml
+    localContent: |
+      apiVersion: v1
+      kind: List
+      items:
+       - 
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          name: ws
+        spec:
+          containers:
+           - 
+            name: theia-dev
+            image: eclipse/che-theia-dev:nightly
+            resources:
+              limits:
+                memory: 2048Mi
+            volumeMounts:
+             - 
+              mountPath: /projects
+              name: projects
+          volumes:
+           - 
+            name: projects
+            persistentVolumeClaim:
+              claimName: projects
+       - 
+        apiVersion: v1
+        kind: PersistentVolumeClaim
+        metadata:
+          name: projects
+        spec:
+          accessModes:
+           - ReadWriteOnce
+          resources:
+            requests:
+              storage: 2Gi
+commands:
+  - name: build
+    actions:
+      - type: exec
+        tool: theia-dev
+        command: cd ${CHE_PROJECTS_ROOT}/che-theia-samples/samples/hello-world-frontend-plugin && yarn
```

Comment:
Kubernetes uses `$(VAR)` convention for variable expansion. IMHO we should try to be as close to Kubernetes in the devfile if for nothing else then for the fact that Kubernetes/Openshift is our target infrastructure.

Reply:
That is just a general remark not related to this PR actually. I will create a separate issue to discuss this.

Reply:
But it is not Che convention, it's a bare command that will be executed in shell
and $(VAR) won't work If I understand correctly
![Screenshot_20190312_131209](https://user-images.githubusercontent.com/5887312/54195925-9d9d7200-44c8-11e9-9712-e419d5a0ca71.png)

Reply:
Hmm... yeah, I didn't notice that this is actually a command to be executed, not something that should be expanded by Kubernetes before it is applied to the container.